src/parser.yy

```diff
@@ -722,7 +737,7 @@ ident:
                 ;
 
 raw_ident:
```

Comment:
how about just removing `raw_ident` all together then you can put `map %prec LOW` and `IDENT %prec LOW` next to each other in `primary_expr`?

Reply:
Sure, I will do this.